Templates/CDS-CSharp/run.csx

```diff
@@ -0,0 +1,66 @@
+﻿#load ".\SecurityHelper.csx"
```

Comment:
Sorry for creeping (agile) spec but we need the following as a comment here to cover our legal requirements for SDK.

// The contents of this template are subject to license terms of the Microsoft Common 
// Data Service Software Development Kit (https://go.microsoft.com/fwlink/?linkid=842862).